server/game/cards/06-TLS/MichaelTheBadgerDodge.js

```diff
@@ -0,0 +1,75 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+
+/** @typedef {import('../../AbilityDsl')} AbilityDsl */
+
+class MichaelTheBadgerDodge extends DudeCard {
+    /** @param {AbilityDsl} ability */
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            targetController: 'current',
+            condition: () => this.hasAttachmentWithKeywords(['melee', 'weapon']), 
+            match: this,
+            effect: [
+                ability.effects.setAsStud()
+            ]
+        });
+        this.action({
+            title: 'Michael "The Badger" Dodge',
+            playType: ['shootout'],
+            cost: [
+                ability.costs.bootSelf(),
+                // Not strictly a cost, but needed so we can reference the result in the 
+                // action handler.
+                ability.costs.pull()
+            ],
+            target: {
+                activePromptTitle: 'Choose an opposing Dude',
+                choosingPlayer: 'current',
+                cardCondition: { 
+                    inOpposingPosse: true, 
+                    controller: 'opponent',
+                    condition: card => !card.booted
+                },
+                cardType: ['dude'],
+                gameAction: 'boot'
```

Comment:
The `gameAction` in the target is used mostly to check if the target can be selected (meaning if that action can be performed on the target), and if subsequently the ability can be triggered.
This would be ok with the old rules as the first sentence was a requirement, so if you could not boot the target, you would not be able to trigger the ability.
However with new rules, it would not be required, as there is that "do as much as you can". So it would be possible to not boot it, but just give it -3 bullets based on the pull.

Reply:
Forgot about the rule change, thanks!